src/components/Form.js

```diff
@@ -0,0 +1,12 @@
+const Form = () => (
+  <section className="absolute self-center mx-auto w-10/12 bottom-0 pt-4 mb-12 border-t-2 border-slate-200">
+    <h2 className="text-slate-500 text-xl font-bold mb-2">ADD NEW BOOK</h2>
+    <form className="w-full h-10 flex items-stretch justfiy-between">
+      <input className="w-3/5 p-3 rounded-md text-slate-600 transition-all h-full block outline-1 outline-blue-100 focus:outline-blue-300" type="text" placeholder="Book title" />
+      <input className="w-1/3 p-3 rounded-md text-slate-600 transition-all h-full block ml-4 outline-blue-100 focus:outline-blue-300" type="" placeholder="Category" />
+      <button className="w-1/5 p-3 flex items-center justify-center rounded-md h-full block bg-blue-600 text-white font-semibold ml-4" type="submit">ADD BOOK</button>
+    </form>
```

Comment:
- [ ] Please, consider that this component should render a form with inputs for the title and the author of the book. Here you have title and category, not the author - I know you can easily fix it 👍

Reply:
@elmar8287, I saw on the template it was category, but I can change this no problem